home/src/main/resources/rdf/i18n/en_CA/interface-i18n/firsttime/vivo_UiLabel.ttl

```diff
@@ -6266,3 +6266,17 @@ uil-data:degree_candidacy.VIVO
         uil:hasApp      "VIVO" ;
         uil:hasKey      "degree_candidacy" ;
         uil:hasPackage  "VIVO-languages" .
+
+uil-data:vis_coauthor_vcard.VIVO
+        rdf:type         owl:NamedIndividual ;
+        rdf:type         uil:UILabel ;
+        rdfs:label       "Include external co-authors"@en-CA ;
+        uil:hasApp       "VIVO" ;
+        uil:hasKey       "vis_coauthor_vcard" .
+
+uil-data:select_vcard_or_person.VIVO
+        rdf:type         owl:NamedIndividual ;
+        rdf:type         uil:UILabel ;
+        rdfs:label       "Please select VCARD only, when the person doesn't have and is not supposed to have a profile in VIVO. Otherwise choose the type \"Person\"."@en-CA ;
```

Comment:
Maybe we can use here: 

Please select external co-author, when the person doesn't have and is not supposed to have a profile in VIVO. Otherwise choose the type \"Person\"."

Reply:
Maybe. In that case VCARD option should be changed accordingly on entry form pages. In any case I think we should change all the translations altogether when there is an agreement on how it should look like to avoid changing the same thing multiple times. It was already changed once in accordance with this comment https://github.com/vivo-project/VIVO/pull/4015#pullrequestreview-2359604501